docs/src/site/markdown/index.md.vm

```diff
@@ -0,0 +1,134 @@
+---
+title: Apache Software Foundation Parent POM
+author: 
+  - Benson Margulies
+  - Hervé Boutemy
+  - Karl Heinz Marbaise
+date: 2016-05-09
+---
+
+<!-- Licensed to the Apache Software Foundation (ASF) under one-->
+<!-- or more contributor license agreements.  See the NOTICE file-->
+<!-- distributed with this work for additional information-->
+<!-- regarding copyright ownership.  The ASF licenses this file-->
+<!-- to you under the Apache License, Version 2.0 (the-->
+<!-- "License"); you may not use this file except in compliance-->
+<!-- with the License.  You may obtain a copy of the License at-->
+<!---->
+<!--   http://www.apache.org/licenses/LICENSE-2.0-->
+<!---->
+<!-- Unless required by applicable law or agreed to in writing,-->
+<!-- software distributed under the License is distributed on an-->
+<!-- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY-->
+<!-- KIND, either express or implied.  See the License for the-->
+<!-- specific language governing permissions and limitations-->
+<!-- under the License.-->
+
+# Apache Software Foundation Parent POM
+
+This project consists of a POM that contains settings that are likely to be useful to any Apache project that is building and releasing code with Maven\. By using this project as a parent, a project gets these settings\.
+
+Most projects will want to override some of these settings\.
+
+The following is a list of the items configured in this POM:
+
+- Top\-level **project properties**:
```

Comment:
do we need all backslashes ...

Reply:
no, but this file and all escape characters are mostly autogenerated from MarkdownSink, which generates them as it is much harder to only use escape characters when really needed. However they don’t do any harm except for making the markdown harder to read.